x/params/proposal_handler_test.go

```diff
@@ -61,23 +69,23 @@ func (suite *HandlerTestSuite) TestProposalHandler() {
 			func() {},
 			true,
 		},
-		{
-			"omit empty fields",
-			testProposal(proposal.ParamChange{
-				Subspace: govtypes.ModuleName,
-				Key:      string(govv1.ParamStoreKeyDepositParams),
-				Value:    `{"min_deposit": [{"denom": "uatom","amount": "64000000"}], "max_deposit_period": "172800000000000"}`,
-			}),
-			func() {
-				depositParams := suite.app.GovKeeper.GetDepositParams(suite.ctx)
-				defaultPeriod := govv1.DefaultPeriod
-				suite.Require().Equal(govv1.DepositParams{
-					MinDeposit:       sdk.NewCoins(sdk.NewCoin("uatom", sdk.NewInt(64000000))),
-					MaxDepositPeriod: &defaultPeriod,
-				}, depositParams)
-			},
-			false,
-		},
+		// {
```

Comment:
@aaronc Shouldn't we use another `app.yaml` for this specific test? I feel like the default `app.yaml` from `testutil` should be the minimal modules required for the tested specific module. Here, gov does not fall in this category.

Reply:
What I meant is from my understanding the app.yaml of a module should be minimalistic. That means only requiring modules that are dependencies or strictly required for running the module app.

Here we need to add as well the gov module which isn't a dependency of params.